### ISSUE TITLE
[Bug] Eraser tool highlights unintuitive and confusing

### ISSUE THREAD

**Issue by T8RIN** [pastthepixels/FreePaint#4]
When i change between paint and eraser this one happens
 
https://github.com/pastthepixels/FreePaint/assets/52178347/aed2f690-c316-42b0-b71d-76396257e159

**Comment by pastthepixels** [comment]
Yeah, that's actually not the stroke width of the drawing! When you draw a closed shape, FreePaint draws a fill first, then a stroke (so the stroke is on top of the fill). Then, when you switch to the eraser tool, FreePaint highlights closed shapes by drawing their fill to be green.
The reason I did this was mainly because this was a quick project and I needed to get it done before a deadline, but the UX definitely needs to be improved in future updates. I'm open to suggestions -- we could even remove the green overlays created by the eraser tool.
For that reason I'll leave this issue up (with a bit of renaming of course) because the interface of the eraser tool can cause a lot of confusion. Thanks for opening this issue!

TL;DR Right now it isn't the stroke width changing, the eraser tool just highlights filled shapes with the color green (and strokes with green too) to let you know you can erase them).

**Comment by T8RIN** [comment]
Okay :)

**Comment by pastthepixels** [comment]
After a bit of work I decided that a full reform of what I wanted to do with the eraser tool would require a reform of a bunch of FreePaint, so to push a reasonable fix beforehand I decided to change the colors of the eraser tool so it would be more obvious what it actually does:

https://github.com/pastthepixels/FreePaint/assets/52388215/22b3bd3a-6568-4bfa-b16d-26d89a73e251

In later versions I'll make the eraser tool function more like a pen and less like a lasso (as someone put it), eliminating the need to highlight paths in green in the first place. For now though, let me know what you think, and if it all looks good I'll merge it for FreePaint 1.1!

**Comment by T8RIN** [comment]
I think that's looks nice, thanks! :)

**Comment by pastthepixels** [comment]
Cool, I'll merge it then!

**Comment by T8RIN** [comment]
Thanks ;)
